packages/flutter/test/widgets/system_context_menu_test.dart

```diff
@@ -699,6 +701,28 @@ void main() {
     },
   );
 
+  test(
+    'can get the IOSSystemContextMenuItemData representation of an IOSSystemContextMenuItemLiveText',
+    () {
+      const IOSSystemContextMenuItemLiveText item = IOSSystemContextMenuItemLiveText();
+      const WidgetsLocalizations localizations = DefaultWidgetsLocalizations();
+      final IOSSystemContextMenuItemData data = item.getData(localizations);
+      expect(data, isA<IOSSystemContextMenuItemDataLiveText>());
+    },
+  );
+
+  test('IOSSystemContextMenuItemDataLiveText debugFillProperties', () {
+    const IOSSystemContextMenuItemDataLiveText item = IOSSystemContextMenuItemDataLiveText();
+    final List<DiagnosticsNode> diagnosticsNodes = item.toDiagnosticsNode().getProperties();
+    expect(diagnosticsNodes, hasLength(0));
+  });
```

Comment:
Also this wouldn't be necessary anymore if you remove Diagnosticable from the class.